manifests/0000_90_cluster-kube-controller-manager-operator_03_servicemonitor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-kube-controller-manager-operator
+  namespace: openshift-cluster-kube-controller-manager-operator
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
```

Comment:
do we need this in controller manager?

Reply:
> do we need this in controller manager?

Because we're based on generic apiserver, I think we do.